### PR TITLE
Update sphinx filter list

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

File: docs/source/filters.rst [hunk 1]
```diff
@@ -109,7 +111,7 @@ SimpleITK Filters
    * - `ClosingByReconstructionImageFilter <https://www.itk.org/SimpleITKDoxygen/html/classitk_1_1simple_1_1ClosingByReconstructionImageFilter.html>`_
      - Closing by reconstruction of an image.
    * - `CoherenceEnhancingDiffusionImageFilter <https://www.itk.org/SimpleITKDoxygen/html/classitk_1_1simple_1_1CoherenceEnhancingDiffusionImageFilter.html>`_
-     - Coherence enhancing diffusion and edge enhancing diffusion.
+     - Coherence enhanging diffusion and edge enhancing diffusion.
```
Comment (zivy) [inline docs/source/filters.rst]: "enhanging": revert the change.

File: docs/source/filters.rst [hunk 2]
```diff
@@ -219,7 +221,7 @@ SimpleITK Filters
    * - `GradientRecursiveGaussianImageFilter <https://www.itk.org/SimpleITKDoxygen/html/classitk_1_1simple_1_1GradientRecursiveGaussianImageFilter.html>`_
      - Computes the gradient of an image by convolution with the first derivative of a Gaussian.
    * - `GrayscaleConnectedClosingImageFilter <https://www.itk.org/SimpleITKDoxygen/html/classitk_1_1simple_1_1GrayscaleConnectedClosingImageFilter.html>`_
-     - Enhance pixels associated with a dark object (identified by a seed pixel) where the dark object is surrounded by a brighter object.
+     - Enhance pixels associated with a dark object (identified by a seed pixel) where the dark object is surrounded by a brigher object.
```
Comment (zivy) [inline docs/source/filters.rst]: "brigher": revert the change .

File: docs/source/filters.rst [hunk 3]
```diff
@@ -389,7 +391,7 @@ SimpleITK Filters
    * - `MorphologicalWatershedFromMarkersImageFilter <https://www.itk.org/SimpleITKDoxygen/html/classitk_1_1simple_1_1MorphologicalWatershedFromMarkersImageFilter.html>`_
      - Morphological watershed transform from markers.
    * - `MorphologicalWatershedImageFilter <https://www.itk.org/SimpleITKDoxygen/html/classitk_1_1simple_1_1MorphologicalWatershedImageFilter.html>`_
-     - Watershed segmentation implementation with morphological operators.
+     - Watershed segmentation implementation with morphogical operators.
```
Comment (zivy) [inline docs/source/filters.rst]: revert this change original was correct.

**Review by zivy — COMMENTED** [review]
Revert some changes.

**Comment by blowekamp** [comment]
Thanks for catching that the script was run with the old documentation. It has been updated now.

**Review by zivy — APPROVED** [review]
